src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md

```diff
@@ -75,7 +75,7 @@ Update an existing blueprint assignment through an assignment file. The format o
 
 ### Example 3
 ```powershell
-$blueprintObject =  Get-AzBlueprint -ManagementGroup "myManagementGroup" -Name "myBlueprintName"
+$blueprintObject =  Get-AzBlueprint -ManagementGroupId "myManagementGroupId" -Name "myBlueprintName"
 Set-AzBlueprintAssignment -Name "myAssignment" -Blueprint $blueprintObject -ManagementGroupId "myManagementGroup" -SubscriptionId 00000000-1111-0000-1111-000000000000 -Location "West US" -Parameter @{P1="v1"; P2="v2"}
```

Comment:
Why "myManagementGroupId"?

Reply:
Reference Get-Blueprint.md. There is no parameter named ManagementGroup, and its examples are using commonly ManagmentGroupId.